src/observations/price_observation.py

```
@@ -0,0 +1,55 @@
+    def get_observation(self, env: PriceEnvironment) -> np.ndarray:
+        try:
+            data = env.get_current_data()
+            min_price, max_price = self.get_price_range(env)
+
+            # Normalize the price
+            normalized_price = (data[self.column_name] - min_price) / (
+                max_price - min_price
+            )
+
+            return np.array([normalized_price], dtype=np.float32)
+        except:
```

Ruff

Do not use bare `except`

(E722)